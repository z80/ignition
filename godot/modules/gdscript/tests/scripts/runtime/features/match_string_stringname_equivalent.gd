# https://github.com/godotengine/godot/issues/60145

func test():
	match "abc":
		&"abc":
			print("String matched StringName")
		_:
			print("no match")

	match &"abc":
		"abc":
			print("StringName matched String")
		_:
			print("no match")
