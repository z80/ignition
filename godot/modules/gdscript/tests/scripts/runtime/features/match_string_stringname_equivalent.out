GDTEST_OK
String matched StringName
StringName matched String
